src/client/pages/Offer/Introduction/HeroOfferDetails.tsx

```diff
@@ -132,7 +155,7 @@ export const HeroOfferDetails: React.FC<Props> = ({
           {numberCoInsured > 0 && ` +${numberCoInsured}`}
         </NameAndCoInsured>
 
-        {address && <Address>{address}</Address>}
+        {address && <QuoteAddress>{address}</QuoteAddress>}
```

Comment:
🤔 Empty strings are one of the [Javascript Falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) and In general, I think empty strings are a good way to indicate 'nothing to display' since types became simpler: `string` rather than `string | null`. The exception would be scenarios where we want to do something in case that variable equals to `null`, which not seems to be the case. With that said, I'm also totally fine to change it to `string | null`.

Maybe someone else can 'untie' this decision 🤣